### ISSUE TITLE
Kolbot Sonic error. General not defined

### ISSUE THREAD

**Issue by zinba1** [kolton/d2bot-with-kolbot#2041]
Bot loads joins server, creates game and drops to and id scroll and gets an error. Can't figure it out
![FB_IMG_1576195882553](https://user-images.githubusercontent.com/58833552/70761772-650f1b00-1d1c-11ea-83ee-83563a03ecf4.jpg)

**Comment by mf022** [comment]
did you see https://github.com/SetupSonic/SetupSonic/wiki/D2Bot%23-Configuration

**Comment by leoallil** [comment]
I'm having the same issue
Did you manage to solve this?
Thanks!

**Comment by lennaertje2** [comment]
> I'm having the same issue
> Did you manage to solve this?
> Thanks!

Found the solution if still need

**Comment by Daggui** [comment]
> 
> 
> > I'm having the same issue
> > Did you manage to solve this?
> > Thanks!
> 
> Found the solution if still need
Can you explain the solution ? thanks

**Comment by lennaertje2** [comment]
> > > I'm having the same issue
> > > Did you manage to solve this?
> > > Thanks!
> > 
> > 
> > Found the solution if still need
> > Can you explain the solution ? thanks

Yes i can! But i will not at this moment i first want you to do the following as it will help you in the long run (as it did to me).

First download a complete new file to have the basic stuff ready, check the guide how to set it up and read carefully especially the profile part where your error is originating from.

**Comment by Daggui** [comment]
i will follow step by step the instruction and come back to you soon then ;)

**Comment by lennaertje2** [comment]
> i will follow step by step the instruction and come back to you soon then ;)

did u succeed ? :)

**Comment by Daggui** [comment]
nop i clear all my files, reinstall , and now i got this
![sonic](https://user-images.githubusercontent.com/66337082/83780136-0ef8b000-a68d-11ea-8292-81e52c89e6b3.jpg)

**Comment by thecrumblord** [comment]
Im getting this same issue pictured above....Any fix?  thank you! (error 222)
